### ISSUE TITLE
BMC is now covered by Springer list

### ISSUE THREAD

**Issue by zuphilip** [citation-style-language/journals#6]
--> MOVED from https://github.com/citation-style-language/utilities/issues/34

<hr/>

Okay, I checked BMC: all active BMC journals are covered with the new list except two which I will add to the extra together with Insectes Sociaux in [#32](https://github.com/citation-style-language/utilities/pull/32). Some have different parents but it seems that the information from this Springer list is more accurate resp. up-to-date.

Should we delete the `bmc` folder or just mentioned that it is not necessary anymore?

Some BMC journals ceased publications:
- aquatic-biosystems.csl --> "no longer receives submissions" http://aquaticbiosystems.biomedcentral.com/
- biology-of-mood-and-anxiety-disorders.csl --> "no longer receives submissions" http://biolmoodanxietydisord.biomedcentral.com/
- bmc-medical-physics.csl --> "ceased to be published by BioMed Central in 2015" http://bmcmedphys.biomedcentral.com/
- cough.csl --> "no longer receives submissions" http://coughjournal.biomedcentral.com/
- genome-integrity.csl --> "ceased to be published by BioMed Central as of June 2014" http://genomeintegrity.biomedcentral.com/
- herpesviridae.csl --> "no longer receives submissions" http://herpesviridae.biomedcentral.com/
- journal-of-clinical-bioinformatics.csl --> "no longer receives submissions" http://jclinbioinformatics.biomedcentral.com/
- journal-of-systems-chemistry.csl --> "was discontinued in 2015 and no longer receives submissions" http://jsystchem.springeropen.com/
- longevity-and-healthspan.csl --> "no longer receives submissions" http://longevityandhealthspan.biomedcentral.com/
- microbial-informatics-and-experimentation.csl --> "no longer receives submissions" http://microbialinformaticsj.biomedcentral.com/
- the-hugo-journal.csl --> "ceased to be published with SpringerOpen" http://thehugojournal.springeropen.com/

Other journals changed the publisher:
- international-archives-of-medicine.csl --> Wikipedia: "It was established in 2008 and published by BioMed Central until the end of 2014. Starting in 2015, the journal is being published by iMed.pub" http://www.intarchmed.com/
- journal-of-brachial-plexus-and-peripheral-nerve-injury.csl --> http://www.thieme.com/books-main/orthopaedic-surgery/product/2201-journal-of-brachial-plexus-and-peripheral-nerve-injury
- journal-of-molecular-signaling.csl --> "The Editors are delighted to announce the re-launch of Journal of Molecular Signaling in partnership with a new publisher. After eight successful years with Biomed Central, from January 2015 the journal will be published with Ubiquity Press." http://www.jmolecularsignaling.com/

Unclear:
- bmc-proceedings.csl --> ? http://bmcproc.biomedcentral.com/

What should happen to these CSL files?

**Comment by adam3smith** [comment]
definitely delete all discountinued styles. I'd suggest also deleting "wrong" ones and creating tickets for them.

I'm also in favor of deleting the bmc folder.  @rmzelle ?

**Comment by rmzelle** [comment]
Yes to all suggestions. (not sure it's necessary to create tickets for all journals that moved away from BMC, though)

**Comment by zuphilip** [comment]
Okay, I will do these steps then (this is mostly a reminder for myself):
 - [ ] Delete all 15 journals from BMC which ceased publication or changed to another publisher in https://github.com/citation-style-language/styles/tree/master/dependent
 - [ ] List the 11 journals which ceased publications in [renamed-styles.json](https://github.com/citation-style-language/styles/blob/master/renamed-styles.json) (Right?)
 - [ ] Delete the bmc folder in https://github.com/citation-style-language/journals/

Moreover, these journals changed the publisher and we may want to deal with that further:
 - [ ] international-archives-of-medicine.csl --> Wikipedia: "It was established in 2008 and published by BioMed Central until the end of 2014. Starting in 2015, the journal is being published by iMed.pub" http://www.intarchmed.com/
 - [ ] journal-of-brachial-plexus-and-peripheral-nerve-injury.csl --> http://www.thieme.com/books-main/orthopaedic-surgery/product/2201-journal-of-brachial-plexus-and-peripheral-nerve-injury 
 - [ ] journal-of-molecular-signaling.csl --> "The Editors are delighted to announce the re-launch of Journal of Molecular Signaling in partnership with a new publisher. After eight successful years with Biomed Central, from January 2015 the journal will be published with Ubiquity Press." http://www.jmolecularsignaling.com/ 
 - [ ] bmc-proceedings.csl --> ? http://bmcproc.biomedcentral.com/

**Comment by adam3smith** [comment]
> List the 11 journals which ceased publications in renamed-styles.json (Right?)
no, we agreed on the Elsevier thread that that's not necessary for dependent styles.

**Comment by rmzelle** [comment]
We could bring it up on xbiblio-devel, but yeah, I think it's only really necessary to include styles identifiers in renamed-styles.json for journals that either change title (it happens) or for which we changed the title/ID.

**Comment by zuphilip** [comment]
The last one "[Journal of molecular signaling](http://www.jmolecularsignaling.com/)" uses IMO a strange [reference style](http://www.jmolecularsignaling.com/about/submissions/#References):

Journal Articles:

1. Fletcher D, Wagstaff CRD. Organisational psychology in elite sport: its emergence, application and future. Psychol Sport Exerc. 2009;10(4):427-34. doi:10.1016/j.psychsport.2009.03.009
2. Sanger F, Nicklen S, Coulson AR (1977) DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci U S A 74: 5463-5467.


Thus, the position of the year depends on whether the journal article has a doi or not??

Do you have seen anything similar before?

**Comment by adam3smith** [comment]
I'm pretty sure they don't mean that. I looked at a couple of examples, and they have the first style in the PDFs (so I'd go with that) and the 2nd one in the html version (of the same article). I don't understand whay, but I'd just go with 1., which has the additional advantage of being pretty standard Vancouver.

**Comment by zuphilip** [comment]
Hm... The most current article from them http://www.jmolecularsignaling.com/articles/10.5334/1750-2187-11-1/galley/268/download/ is even using some slight variants (bold italics, commas, spaces) to the guidelines.

Actually, I don't want to spend more time on this at the moment and maybe they will update and clarify their rules in the future. Thus, I will left the Journal of molecular signaling without a CSL style for the moment.

**Comment by damnation333** [comment]
I've just contacted the editors for Journal of Molecular Signaling as their author guidelines and also citation style in various publications still seems messy. Let's see if/what they answer. If not I'll put something together that seems like the common denominator between most of their publications?

**Comment by POBrien333** [comment]
HI gang,
Just looking at this request here and it looks like health-research-policy-and-systems just follows the "old" biomed-central style and not one of the Springer styles: https://health-policy-systems.biomedcentral.com/submission-guidelines/preparing-your-manuscript and for an article: https://health-policy-systems.biomedcentral.com/articles

Shall I just make a dependent for that?

**Comment by POBrien333** [comment]
Is anything outstanding from this issue or can this be closed?
